docs/user/exams/live_statistics.rst

```diff
@@ -0,0 +1,129 @@
+.. _live_statistics:
+
+***************
+
+1. Enable/Disable
+=================
+
+Artemis provides several ways to enable the exam live statistic dashboard. Instructors can enable the exam live statistics dashboard on the exam creation page.
+
+.. figure:: instructor/exam_configuration_enable_exam_live_statistics.png
+   :alt: Checkbox to enable the exam live statistics
+   :align: center
+
+   Checkbox to enable the exam live statistics during the exam configuration
+
+In the exam live statistics dashboard, instructors can see the current status of this feature. In addition, they can then deactivate or activate it.
+
+.. figure:: instructor/exam_live_statistics_enabled.png
+   :alt: Exam live statistics enabled
+   :align: center
+
+   The exam live statistics status indicator indicates that the exam live statistics is enabled
```

Comment:
"indicator indicates" sounds redundant. Maybe you could use related/synonyms words. What do you think?